torch/ao/quantization/_dbr/auto_trace.py

```diff
@@ -85,11 +85,14 @@ class QuantizationPrepareTensorProxy(torch.Tensor):
         @classmethod
         def __torch_function__(cls, func, types, args=(), kwargs=None):
             nonlocal global_disable_torch_function_override
-            if global_disable_torch_function_override:
-                return super().__torch_function__(func, types, args, kwargs)
-
-            if func == torch.Tensor.__repr__:
+            if (
```

Comment:
should we define a helper function

Reply:
maybe if n gets from 2 to 3?  going to keep it for now but will keep an eye for standardizing this later, hope that's ok?

Reply:
sure, sounds good